README.md

```diff
@@ -1,10 +1,145 @@
+    - [_REACT_APP_XIS_CATALOGS_API_](#react-app-xis-catalogs-api)
+    - [_REACT_APP_XIS_COMPOSITELEDGER_API_](#react-app-xis-compositeledger-api)
+  - [Creating a local env](#creating-a-local-environment-file)
+    - [_Walk through_](#walk-through)
+    - [_Template_](#env-template)
+  - [Important Notes](#important-notes)
+- [**Available Scripts**](#available-scripts)
+  - [yarn start](#yarn-start)
+  - [yarn test](#yarn-test)
+  - [yarn build](#yarn-build)
+  - [yarn coverage](#yarn-coverage)
+
+---
+
+## Installation
+
+### Clone Project
+
+```bash
+git clone git@github.com:OpenLXP/openlxp-xms-ui.git
+```
+
+### Install Project Dependencies
+
+Start off by verifying that you have `yarn` installed.
+
+```ps1
+yarn -version
+```
+
+If `yarn` is not installed use the following command to install it. This will install the `yarn` package manager globally on your system.
+
+_**global install**_
+
+```ps1
+npm install yarn -g
+```
+
+_**local install**_
+
+```ps1
+npm install yarn
+```
+
+Once yarn has been installed you will need to install the project dependencies. Using the following command we will manually set your yarn version for this project.
+
+```ps1
+yarn set version 1.22.1
+```
+
+After the version has been installed and set we will install the dependencies. Using the following command we will install all the project dependencies.
+
+```bash
+yarn install package.json
+```
+
+---
+
+## Getting started
+
+### Environment Variables
+
+This project makes use of globally available environment variables. Below are the required environment variables required for this project.
+
+#### **REACT_APP_XIS_CATALOGS_API**
+
+This is the root API endpoint used by the application to access XIS catalogs.
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/
+```
+
+#### **REACT_APP_XIS_COMPOSITELEDGER_API**
+
+This is the endpoint for accessing the XIS comositeledger.
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/es-api/
+```
+
+### Creating a local environment file
+
+#### **Walk through**
+
+Let's create a local `.env.local` file. If you are in a code editor you can right click and create new file.
+
+![vscode create new file](./readme-assets/vscode-create-new-file.png)
+
+If you are using the terminal use the following command to create a new file.
+
+_**bash command**_
+
+```bash
+touch <PATH_TO_YOUR_PROJECT_ROOT>/.env.local
+```
+
+_**powershell**_
+
+```ps1
+New-Item -Path <PATH_TO_YOUR_PROJECT_ROOT>\.env.local -ItemType File
+```
+
+Navigate to the newly created file and paste the template (_below_) into the file. Replace `<YOUR_BACKEND_ENDPOINT>` with your localhost setup **or** your live endpoint.
+
+#### **.env Template**
+
+```text
+REACT_APP_XIS_CATALOGS_API=<YOUR_BACKEND_ENDPOINT>/api/catalogs/
+REACT_APP_XIS_COMPOSITELEDGER_API=<YOUR_BACKEND_ENDPOINT>/api/metadata/
+```
+
+### Important Notes
+
+To use this piece of code without any issues you will need the XDS component to accompany it.
+
+[OpenLXP XMS](https://github.com/OpenLXP/openlxp-xms)
```

Comment:
Just checking did you mean to tag XMS here?

Reply:
Ill get that, Thanks!

Reply:
Done